app/controllers/defendants_controller.rb

```diff
@@ -91,6 +95,10 @@ def unlink_laa_reference_and_redirect
   rescue CourtDataAdaptor::Errors::InternalServerError,
          CourtDataAdaptor::Errors::ClientError => e
     handle_error(e, I18n.t('defendants.unlink.failure'), I18n.t('error.it_helpdesk'))
+  rescue StandardError => e
+    logger.error "MAAT_UNLINK_ERROR: #{e.message}"
```

Comment:
I think it would make more sense to call `handle_error(e, I18n.t('defendants.unlink.unknown_error'), I18n.t('error.it_helpdesk'))` here. That way:
a) we still log the error to Sentry, so we'll get an alert about it, and
b) if by any chance the error has a cda error code in it, we still have a chance of showing a message specific to that error code.

Reply:
I tried, but  handle_error is only compatible with `CourtDataAdaptor::Errors`, not with StandardError.
Anyway I agree on capturing the exception on Sentry.